services/movies.tsx

```diff
@@ -1,3 +1,5 @@
+const fetch = require('node-fetch');
```

Comment:
T'en a pas besoin normalement

Reply:
Si je ne le définis pas comme ça, l'app marche normalement mais les tests ne passent pas, j'ai `ReferenceError: fetch is not defined`